tests/functional/api/basic_lti_launch.py

```diff
@@ -99,7 +98,41 @@ def module_item_configuration(self, request, db_session):
 
     @pytest.fixture
     def lti_params(self):
-        return self.json_fixture("lti_params/good_params.json")
+        return {
+            "context_id": "con-182",
+            "context_label": "SI182",
+            "context_title": "Design of Personal Environments",
+            "context_type": "CourseSection",
+            "custom_context_memberships_url": "https://apps.imsglobal.org/lti/cert/tp/tp_membership.php/context/con-182/membership?b64=a2puNjk3b3E5YTQ3Z28wZDRnbW5xYzZyYjU%3D",
+            "custom_context_setting_url": "https://apps.imsglobal.org/lti/cert/tp/tp_settings.php/lis/CourseSection/con-182/bindings/ims/cert/custom?b64=a2puNjk3b3E5YTQ3Z28wZDRnbW5xYzZyYjU%3D",
+            "custom_link_setting_url": "$LtiLink.custom.url",
+            "custom_system_setting_url": "https://apps.imsglobal.org/lti/cert/tp/tp_settings.php/ToolProxy/Hypothesis1b40eafba184a131307049e01e9c147d/custom?b64=a2puNjk3b3E5YTQ3Z28wZDRnbW5xYzZyYjU%3D",
+            "custom_tc_profile_url": "https://apps.imsglobal.org/lti/cert/tp/tp_tcprofile.php?b64=a2puNjk3b3E5YTQ3Z28wZDRnbW5xYzZyYjU%3D",
+            "launch_presentation_document_target": "iframe",
+            "launch_presentation_locale": "en_US",
+            "launch_presentation_return_url": "https://apps.imsglobal.org/lti/cert/tp/tp_return.php/basic-lti-launch-request",
+            "lis_course_section_sourcedid": "id-182",
+            "lis_person_contact_email_primary": "jane@school.edu",
+            "lis_person_name_family": "Lastname",
+            "lis_person_name_full": "Jane Q. Lastname",
+            "lis_person_name_given": "Jane",
+            "lis_person_sourcedid": "school.edu:jane",
+            "lti_message_type": "basic-lti-launch-request",
+            "lti_version": "LTI-1p0",
+            "oauth_callback": "about:blank",
+            "oauth_signature_method": "HMAC-SHA1",
+            "oauth_version": "1.0",
+            "resource_link_id": "rli-1234",
+            "resource_link_title": "Link 1234",
+            "resourcelinkid": "rli-1234",
+            "roles": "Instructor",
+            "tool_consumer_info_product_family_code": "imsglc",
+            "tool_consumer_info_version": "1.1",
+            "tool_consumer_instance_description": "IMS Testing Description",
+            "tool_consumer_instance_guid": "IMS Testing",
+            "tool_consumer_instance_name": "IMS Testing Instance",
+            "user_id": "123456",
+        }
```

Comment:
Having the params directly in the Python file with the code that uses them is a lot better. For example on another branch I was debugging a test that was crashing because it was still trying to use an old value for one of these params. I had changed the value of the param, and updated it everywhere I could find it mentioned in the source code. Eventually I had to track down this unrelated-looking `lti_params()` fixture, the `json_fixture()` method (defined in a base class in another file) and eventually the `good_params.json` file, to find the outdated param value. I still didn't find the value that I wanted because the params from the JSON file also get later post-processed by some Python code later on